src/Config.Binder/ConfigurationBinder.cs

```diff
@@ -179,6 +180,20 @@ private static void BindNonScalar(this IConfiguration configuration, object inst
         {
             if (instance != null)
             {
+                foreach (var requiredProperty in options.RequiredProperties)
+                {
+                    var propertyName = requiredProperty;
+                    if (options.ConfigToPropertyMap.ContainsKey(propertyName) && !string.IsNullOrEmpty(options.ConfigToPropertyMap[propertyName]))
```

Comment:
Should we use mapping to a string/null property name as the mechanism to ignore config properties?

Reply:
And probably might as well add a dedicated `binderOptions.IgnoredProperties` which is checked last to skip property names that are in there.

Reply:
Should we use mapping to a string/null property name as the mechanism to ignore config properties?
I don't think so the behavior could be difficult to understand in this case...
yes the binderOptions.IgnoredProperties is a good idea will implement it